docs/cli/enterprise.md

```diff
@@ -0,0 +1,219 @@
+
+2. Provide the canonical _definition_ for each server on the allowlist.
+
+```json
+{
+  "allowMCPServers": ["corp-data-api", "source-code-analyzer"],
+  "mcpServers": {
+    "corp-data-api": {
+      "command": "/usr/local/bin/start-corp-api.sh",
+      "timeout": 5000
+    },
+    "source-code-analyzer": {
+      "command": "/usr/local/bin/start-analyzer.sh"
+    }
+  }
+}
+```
+
+This pattern is more secure because it uses both definition and an allowlist. Any server a user defines will either be overridden by the system definition (if it has the same name) or blocked because its name is not in the `allowMCPServers` list.
+
+### Less Secure Pattern: Omitting the Allowlist
+
+If the administrator defines the `mcpServers` object but fails to also specify the `allowMCPServers` allowlist, users may add their own servers.
+
+**Example System `settings.json`:**
+
+This configuration defines servers but does not enforce the allowlist.
+The administrator has NOT included the "allowMCPServers" setting.
+
+```json
+{
+  "mcpServers": {
+    "corp-data-api": {
+      "command": "/usr/local/bin/start-corp-api.sh"
+    }
+  }
+}
+```
+
+In this scenario, a user can add their own server in their local `settings.json`. Because there is no `allowMCPServers` list to filter the merged results, the user's server will be added to the list of available tools and allowed to run.
+
+## Enforcing Sandboxing for Security
+
+To mitigate the risk of potentially harmful operations, you can enforce the use of sandboxing for all tool execution. The sandbox isolates tool execution in a containerized environment.
+
+**Example:** Force all tool execution to happen within a Docker sandbox.
+
+```json
+{
+  "sandbox": "docker"
+}
+```
+
+You can also specify a custom, hardened Docker image for the sandbox using the `--sandbox-image` command-line argument or by building a custom `sandbox.Dockerfile` as described in the [Sandboxing documentation](./configuration.md#sandboxing).
+
+## Controlling Network Access via Proxy
+
+In corporate environments with strict network policies, you can configure Gemini CLI to route all outbound traffic through a corporate proxy. This can be set via an environment variable, but it can also be enforced for custom tools via the `mcpServers` configuration.
+
+**Example (for an MCP Server):**
+
+```json
+{
+  "mcpServers": {
+    "proxied-server": {
+      "command": "node",
+      "args": ["mcp_server.js"],
+      "env": {
+        "HTTP_PROXY": "http://proxy.example.com:8080",
+        "HTTPS_PROXY": "http://proxy.example.com:8080"
+      }
+    }
+  }
+}
+```
+
+## Telemetry and Auditing
+
+For auditing and monitoring purposes, you can configure Gemini CLI to send telemetry data to a central location. This allows you to track tool usage and other events.
+
+**Example:** Enable telemetry and send it to a local OTLP collector.
+
+```json
+{
+  "telemetry": {
+    "enabled": true,
+    "target": "local",
```

Comment:
I'd expect enterprises to use GCP and other OpenTelemetry provider, not send to local collector on a user's machine